### ISSUE TITLE
🔙 Add a [Back to balance.io] button

### ISSUE THREAD

**Issue by ricburton** [balance-io/balance-manager#363]
<img width="907" alt="screen shot 2018-07-23 at 21 27 16" src="https://user-images.githubusercontent.com/438160/43111422-311f3e84-8ebf-11e8-97f8-5ba287818d01.png">

I have seen people click around out of curiosity and then get lost.

**Comment by defidavid** [comment]
@ricburton -- I've got a branch I'd like to submit for this.  It appears I don't have permission to do so...

**Comment by ricburton** [comment]
You can make a pull request :)

**Comment by defidavid** [comment]
Looks like I'm still lacking some permissions.

![image](https://user-images.githubusercontent.com/2999380/43559456-bab13664-95d3-11e8-83f0-97b328c58016.png)

**Comment by ricburton** [comment]
Hi David,

I think you are trying to push to our branch. You should be pushing to yours.

Does that make more sense? :)

**Comment by defidavid** [comment]
Not quite, haha.  So I cloned the `latest` branch and then branched off of it.  My branch is, `backButton`, which I have locally.  I was then going to push my branch to `origin` (the github repo) and create a PR against the `latest` branch.

This is the workflow I have always used when using github.  I definitely wouldn't call myself a git expert so I might be missing something :)

**Comment by andyrewlee** [comment]
@TrustyTexan does it work if you fork the repo and then make a pull request?

@ricburton How would this work in a mobile view? 

I am not sure having this button is the best idea because when would a user want to go back to the landing page. I imagine balance manager having a standalone electron application in the future and it wouldn't make sense in that context either.

**Comment by ricburton** [comment]
Thanks for tryin David. Can you explain the problems you are having making the contribution? I found it tricky to begin with too.

Pierrick has been working on making all of Manager responsive.

We have no plans for an electron app. We tried making macOS software in the early days. We find it’s just not something normal people really use a lot.

Richard

**Comment by defidavid** [comment]
@ricburton -- When I attempt to create a PR, I need to select the branch against which to compare the `latest` branch.  The the workflow I'm familiar with requires me to push my local branch to origin (the github repo), so that I can select my branch when creating the PR.  I just need the ability/permission to push my local branch to the github repo, so a PR can be created on the github repo comparing my branch (which needs to be pushed to the github repo) to the `latest` branch.  I am almost positive this is a permission issue on your github repo.

![image](https://user-images.githubusercontent.com/2999380/44855112-f645a580-ac2f-11e8-8ea1-06658f9dec4d.png)

My branch has code that handles responsiveness as follows:

![image](https://user-images.githubusercontent.com/2999380/44855452-ae734e00-ac30-11e8-9fe4-b4d0c525c890.png)

![image](https://user-images.githubusercontent.com/2999380/44855477-bcc16a00-ac30-11e8-8261-31d01c41a0b3.png)

![image](https://user-images.githubusercontent.com/2999380/44855488-c5b23b80-ac30-11e8-957a-75987540c6b7.png)


Help me help you :)


![image](https://user-images.githubusercontent.com/2999380/44855569-ff834200-ac30-11e8-9589-4732e8526a24.png)

**Comment by defidavid** [comment]
Btw, I was on vacation camping in the mountains of Colorado.  I just saw this message yesterday.

**Comment by ricburton** [comment]
@jinchung can you give David some advice here?

Richard

**Comment by defidavid** [comment]
@ricburton @jinchung -- I figured out the issue.  Given that this is the first OSS contribution I've made, I have not needed to fork a repo in the past.  I instead cloned your repo, but I should have forked.  I'll go through that process when I get some time.  I'm sorry for the confusion!
